Networking/Networking/Remote/GenerativeContentRemote.swift

```diff
@@ -199,80 +200,71 @@ private extension GenerativeContentRemote {
-                dateOnSaleEnd: nil,
-                productTypeKey: ProductType.simple.rawValue,
-                statusKey: ProductStatus.published.rawValue,
-                featured: false,
-                catalogVisibilityKey: ProductCatalogVisibility.visible.rawValue,
-                fullDescription: "",
-                shortDescription: "",
-                sku: "",
-                price: "",
-                regularPrice: "",
-                salePrice: "",
-                onSale: false,
-                purchasable: false,
-                totalSales: 0,
-                virtual: false,
-                downloadable: false,
-                downloads: [],
-                downloadLimit: -1,
-                downloadExpiry: -1,
-                buttonText: "",
-                externalURL: "",
-                taxStatusKey: ProductTaxStatus.taxable.rawValue,
-                taxClass: "",
-                manageStock: false,
-                stockQuantity: nil,
-                stockStatusKey: ProductStockStatus.inStock.rawValue,
-                backordersKey: ProductBackordersSetting.notAllowed.rawValue,
-                backordersAllowed: false,
-                backordered: false,
-                soldIndividually: false,
-                weight: "",
-                dimensions: ProductDimensions(length: "", width: "", height: ""),
-                shippingRequired: true,
-                shippingTaxable: true,
-                shippingClass: "",
-                shippingClassID: 0,
-                productShippingClass: nil,
-                reviewsAllowed: true,
-                averageRating: "",
-                ratingCount: 0,
-                relatedIDs: [],
-                upsellIDs: [],
-                crossSellIDs: [],
-                parentID: 0,
-                purchaseNote: "",
-                categories: [],
-                tags: [],
-                images: [],
-                attributes: [],
-                defaultAttributes: [],
-                variations: [],
-                groupedProducts: [],
-                menuOrder: 0,
-                addOns: [],
-                isSampleItem: false,
-                bundleStockStatus: nil,
-                bundleStockQuantity: nil,
-                bundledItems: [],
-                compositeComponents: [],
-                subscription: nil,
-                minAllowedQuantity: nil,
-                maxAllowedQuantity: nil,
-                groupOfQuantity: nil,
-                combineVariationQuantities: nil)
+
+        let tagsAsString = {
+            guard !tags.isEmpty else {
+                return ""
+            }
+
+            return
+                ", tags:Given the list of available tags ```\(tags.map { $0.name }.joined(separator: ", "))```, " +
+                "suggest an array of the best matching tags for this product, if no matches are found return an empty array, " +
+                "don’t suggest any value other than the available ones"
```

Comment:
👀 Please add a period mark at the end of the sentence to make the prompt readable.

Reply:
Done in 1e61884e951eb7fa2832e4e2ecb36c89fa9777a7